File: backend/app/src/main/java/com/david/app/model/Material.java
package com.david.app.model;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "materiales")
@Data
public class Material {
    @Id
    private String id;

    private String nombre;
    private String categoria;
    private Integer stockTotal;
    private Integer stockDisponible;
    private String ubicacionAlmacen;
    private String estado;
}

File: frontend/src/app/model/material.ts
export interface Material {
  id?: string;
  nombre: string;
  categoria: string;
  stockTotal: number;
  stockDisponible: number;
  stockReservado: number;
  stockReparacion: number;
  tarifaDia: number;
  almacen: string;
  ubicacionAlmacen: string;
  estado: string;
}
